crmprtd/wamr/normalize.py

```diff
@@ -46,6 +46,12 @@ def normalize(file_stream):
 
         if unit == '% RH':
             unit = re.sub('% RH', '%', unit)
+        elif unit == '\u00b0C':
```

Comment:
This is a good catch, but most if/elif patterns lend themselves to either a mapping or iteration. E.g.:

```
substitutions = [
    ('% RH', '%'),
    ('mb', 'millibar')
    ...
]
for src, dest in substitutions:
    if unit == src:
        unit = re.sub(src, dest, unit)
```